templates/commit-snippet-2

```diff
@@ -1,7 +1,8 @@
 # Code for checking consistency of @@EBUILD@@
 pushd @@CP@@ > /dev/null
 
-repoman -d full || exit 1
-repoman -m "@@CP@@: ${DESCR}" commit || exit 1
+pkgcheck scan || exit 1
+git add @@EBUILD@@
```

Comment:
Why isn't an error here fatal also? In the end we should remove all these ```|| exit 1``` and just add ```set -e ``` at the beginning of the script.